hybrasyl/Objects/Monster.cs

```diff
@@ -42,6 +42,8 @@ public class Monster : Creature, ICloneable
 
         private Spawn _spawn;
 
+        private MonsterLoot spawnLoot;
```

Comment:
private variables should always begin with an underscore. '_spawnLoot'

Reply:
Understood, will adjust.